test/emulator_backend.go

```diff
@@ -55,30 +55,27 @@ type EmulatorBackend struct {
 	accountKeys map[common.Address]map[string]keyInfo
 
 	// fileResolver is used to resolve local files.
-	//
 	fileResolver FileResolver
 
 	// A property bag to pass various configurations to the backend.
 	// Currently, supports passing address mapping for contracts.
 	configuration *stdlib.Configuration
-}
 
-func (e *EmulatorBackend) StandardLibraryHandler() stdlib.StandardLibraryHandler {
-	// TODO:
-	return nil
+	stdlibHandler stdlib.StandardLibraryHandler
 }
 
 type keyInfo struct {
 	accountKey *sdk.AccountKey
 	signer     crypto.Signer
 }
 
-func NewEmulatorBackend(fileResolver FileResolver) *EmulatorBackend {
+func NewEmulatorBackend(fileResolver FileResolver, stdlibHandler stdlib.StandardLibraryHandler) *EmulatorBackend {
 	return &EmulatorBackend{
-		blockchain:   newBlockchain(),
-		blockOffset:  0,
-		accountKeys:  map[common.Address]map[string]keyInfo{},
-		fileResolver: fileResolver,
+		blockchain:    newBlockchain(),
+		blockOffset:   0,
+		accountKeys:   map[common.Address]map[string]keyInfo{},
+		fileResolver:  fileResolver,
+		stdlibHandler: stdlibHandler,
```

Comment:
Had to set the `stdlibHandler` as a field in the Emulator backend, otherwise if we are to pass it as an argument to the functions, we'll have to change the interface API (in the cadence repo), and update all the downstream dependencies again.